lib/active_fedora/file/attributes.rb

```diff
@@ -49,6 +49,11 @@ def create_date
     created && created.first
   end
 
+  def modified_date
+    modified = metadata.attributes["http://fedora.info/definitions/v4/repository#lastModified"]
+    modified && modified.first
```

Comment:
@atz would fedora ever return more than one value for lastModified? I've never seen this before.